roles/ceph-osd/tasks/docker/start_docker_osd.yml

```diff
@@ -16,6 +16,17 @@
   with_items: "{{ ceph_osd_docker_devices }}"
   register: osd_prepared
 
+- set_fact:
+    docker_env_args: -e OSD_JOURNAL={{ item.1 }}
```

Comment:
From Deepthi Dharwar's review:  The reference to item.1 seems out of scope.  Doesn't this var have to be referenced from a task that has a **with_together** clause?  It seems to be used in the tasks below that do have a with_together clause, and you could just add dedicated_devices|shuffle to the with_together  and reference it that way, right?